brave-lists/debounce.json

```diff
@@ -434,6 +434,16 @@
     "action": "regex-path",
     "param": "^/([^/]+)/s(/.*)$"
   },
+  {
+    "include": [
+      "*://click.pstmrk.it/3s/*"
```

Comment:
There are other variants:
https://click.pstmrk.it/2sm
https://click.pstmrk.it/2s
https://click.pstmrk.it/2m
etc.

This should be `*://click.pstmrk.it/*`